books.c

```diff
@@ -212,6 +216,17 @@ void insertBookCopy(BookCopy* book_copy, BookCopy** head)
     current->next = book_copy;
 }
 
+BookCopy* getAvailableCopyOfBook(Book* book)
+{
+    BookCopy* current = inventory;
+    while (current)
+    {
+        if (!strcmp(current->ISBN, book->ISBN) && current->status == active)
+            return current;
+        current = current->next;
+    }
+}
```

Comment:
Create a variable of type BookCopy (default value to be NULL) and update the value of this variable line 225 and call a break statement. 
Return that variable on line 228